<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2020-09-24T17:05:49 NZST" name="Suite" finished-at="2020-09-24T17:06:44 NZST" duration-ms="54736">
    <groups>
    </groups>
    <test started-at="2020-09-24T17:05:49 NZST" name="HomePagePass" finished-at="2020-09-24T17:06:17 NZST" duration-ms="28493">
      <class name="TP.E2ESelenium.HomePage">
        <test-method is-config="true" signature="setup()[pri:0, instance:TP.E2ESelenium.HomePage@26794848]" started-at="2020-09-24T17:05:49 NZST" name="setup" finished-at="2020-09-24T17:06:08 NZST" duration-ms="18905" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="homepagenaviagation(java.lang.String, java.lang.String)[pri:0, instance:TP.E2ESelenium.HomePage@26794848]" started-at="2020-09-24T17:06:08 NZST" name="homepagenaviagation" data-provider="getData" finished-at="2020-09-24T17:06:15 NZST" duration-ms="6676" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[sele2e@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[seletest]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- homepagenaviagation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:TP.E2ESelenium.HomePage@26794848]" started-at="2020-09-24T17:06:15 NZST" name="tearDown" finished-at="2020-09-24T17:06:17 NZST" duration-ms="2825" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- TP.E2ESelenium.HomePage -->
    </test> <!-- HomePagePass -->
    <test started-at="2020-09-24T17:06:19 NZST" name="ValidationAssertFail" finished-at="2020-09-24T17:06:44 NZST" duration-ms="24894">
      <class name="TP.E2ESelenium.ValidateTest">
        <test-method is-config="true" signature="setup()[pri:0, instance:TP.E2ESelenium.ValidateTest@23348b5d]" started-at="2020-09-24T17:06:19 NZST" name="setup" finished-at="2020-09-24T17:06:35 NZST" duration-ms="16408" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="validateTest(java.lang.String, java.lang.String)[pri:0, instance:TP.E2ESelenium.ValidateTest@23348b5d]" started-at="2020-09-24T17:06:35 NZST" name="validateTest" data-provider="getData" finished-at="2020-09-24T17:06:40 NZST" duration-ms="5032" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[sele]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[seletest]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [There is 1 erro] but found [There is 1 error]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [There is 1 erro] but found [There is 1 error]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at TP.E2ESelenium.ValidateTest.validateTest(ValidateTest.java:37)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:TP.E2ESelenium.ValidateTest@23348b5d]" started-at="2020-09-24T17:06:40 NZST" name="tearDown" finished-at="2020-09-24T17:06:44 NZST" duration-ms="3433" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- TP.E2ESelenium.ValidateTest -->
    </test> <!-- ValidationAssertFail -->
  </suite> <!-- Suite -->
</testng-results>
